### PR TITLE
X axis time display for testing

### PR DESCRIPTION
**Pull request by Khosiyat**

### CODE REVIEW

**Review by Mustafina1990 — APPROVED** [review]
@Khosiyat it`s better that I will resolve conflict, because I worked under last task with GraphLine, so send me please on Slack when it need to do.

**Comment by whuimei** [comment]
Hey team @zrohacova @Mustafina1990 @Khosiyat , is there something missing from the pull request? At the meeting last Mon, all of you spent a lot of time discussing how to improve code review process and agreed to a workflow around pull request commit messages.

File: biologging-sensor-data/package.json
[the remark this answers is not in the document]
Comment (Khosiyat) [reply biologging-sensor-data/package.json:1]: @zrohacova, which packages?

**Comment by Khosiyat** [comment]
> '...so send me please on Slack when it need to do'.

@Mustafina1990, what am I supposed to send?

**Comment by Khosiyat** [comment]
@zrohacova , @Mustafina1990, I [pushed another branch](https://github.com/biodiversitydata-se/biologging-sensor-client/tree/X-Axis-Time-Display-For-Testing-2). Could you please **approve** the review so that Shanon may test before today's meeting?

**Pull request closed without merging**